README.md

```diff
@@ -1,16 +1,15 @@
-# Stardust  [![Code Climate](https://img.shields.io/codeclimate/github/TechnologyAdvice/stardust.svg?style=flat-square)](https://codeclimate.com/github/TechnologyAdvice/stardust) [![Gemnasium](https://img.shields.io/gemnasium/TechnologyAdvice/stardust.svg?style=flat-square)](https://gemnasium.com/TechnologyAdvice/stardust)
+# Stardust  [![Code Climate](https://img.shields.io/codeclimate/github/TechnologyAdvice/stardust.svg?style=flat-square)](https://codeclimate.com/github/TechnologyAdvice/stardust)
+[![Gemnasium](https://img.shields.io/gemnasium/TechnologyAdvice/stardust.svg?style=flat-square)](https://gemnasium.com/TechnologyAdvice/stardust)
+[![Circle CI](https://circleci.com/gh/TechnologyAdvice/stardust/tree/master.svg?style=svg)](https://circleci.com/gh/TechnologyAdvice/stardust/tree/master)
 
-[React](https://facebook.github.io/react/) component library based on [Semantic UI](http://semantic-ui.com/).
+[React] component library based on [Semantic UI].
 
-```
-npm i stardust -S
-```
+    npm i stardust -S
 
 ## Learn
-Check out the [Documentation](https://technologyadvice.github.io/stardust/).
+Check out the [Doc Site] for usage documentation.
```

Comment:
Wow I had no idea you could do that (include the links at the bottom). Sweet :+1: